src/super_gradients/training/models/detection_models/customizable_detector.py

```diff
@@ -182,18 +183,39 @@ def predict(
         conf: Optional[float] = None,
         batch_size: int = 32,
         fuse_model: bool = True,
+        target_bboxes: Optional[List[np.ndarray]] = None,
+        target_bboxes_format: Optional[str] = None,
+        target_class_ids: Optional[List[np.ndarray]] = None,
     ) -> ImagesDetectionPrediction:
         """Predict an image or a list of images.
 
         :param images:      Images to predict.
+
         :param iou:         (Optional) IoU threshold for the nms algorithm. If None, the default value associated to the training is used.
+
         :param conf:        (Optional) Below the confidence threshold, prediction are discarded.
                             If None, the default value associated to the training is used.
         :param batch_size:  Maximum number of images to process at the same time.
+
         :param fuse_model:  If True, create a copy of the model, and fuse some of its layers to increase performance. This increases memory usage.
+
+        :param target_bboxes: Optional[List[np.ndarray]], ground truth bounding boxes. Can either be an np.ndarray of shape
+         (image_i_object_count, 4) when predicting a single image, or a list of length len(target_bboxes), containing such arrays.
+         When not None, will plot the predictions and the ground truth bounding boxes side by side (i.e 2 images stitched as one).
+
+        :param target_class_ids: Optional[List[np.ndarray]], ground truth target class indices. Can either be an np.ndarray of shape
+         (image_i_target_count) when predicting a single image, or a list of length len(target_bboxes), containing such arrays (default=None).
```

Comment:
shouldn't this be "list of length len(images)" ?

Reply:
It's the same length, but I will change it for clarity.